3-extra/1-count-words.js

```diff
@@ -23,12 +23,33 @@
     - Setting values on an object
 */
 
-function countWords(string) {
-  const wordCount = {};
-
-  // write code here
-
-  return wordCount;
+function countWords(string){
+ 
+  let test2S = string.split(" ");
```

Comment:
maybe try and find simpler names for your variables when your can, even if they are longer. Here you could call it `words`.